composer.json

```
@@ -40,6 +40,7 @@
     },
     "require-dev": {
         "facile-it/paraunit": "^1.3 || ^2.0",
+        "infection/infection": "^0.26.6",
```

```suggestion
        "infection/infection": "^0.27",
```

Since we run Infection on PHP8.3 in CI (and remove it completely on any other job), let's bump the minimal version. For local development it also will require 8.1, but we need it anyway because of `dev-tools`.